cinder/README.md

```diff
@@ -214,7 +216,7 @@ unity_storage_pool_names=
 
 ## Multiple Back-ends
 
-`cinder-dellemc-unity-config.yaml` and `cinder-dellemc-vnx-config.yaml` couldn't use to configure multiple backends.
+`cinder-dellemc-unity-config.yaml` and `cinder-dellemc-vnx-config.yaml` cannot be usedto configure multiple backends.
```

Comment:
typo 'usedto' , should be 'used to'

Reply:
Why we cannot use them to configure multiple backends?
Actually we can use them to configure multiple backends.